TAScheduler/views.py

```diff
@@ -342,15 +388,30 @@ def post(self, request):
         print(request.session["user"])
         try:
             determineUser(request.session["user"]).createUser(account_info, role=request.POST["role"])
-            return render(request, "success.html", {"message": "User successfully created",
-                                                    "previous_url": "/home/manageaccount/create"})
+            users = list(map(str, Administrator.objects.all()))
+            users.extend(list(map(str, Instructor.objects.all())))
+            users.extend(list(map(str, TA.objects.all())))
+            return render(request, "accountManagement/account_management.html",
+                          {"users": users, "current_user": request.session.get("user"),
+                           "message": "Successfully created account"})
         except Exception as e:
-            return render(request, "error.html", {"message": e,
-                                                  "previous_url": "/home/manageaccount/create"})
+            users = list(map(str, Administrator.objects.all()))
```

Comment:
so are we getting rid of the success/error htmls? if so why

Reply:
By getting rid of the success and error pages we have one less page the user needs to navigate through. If there is an error let the user fix it on the page they were using. If it was a success let the user know but put them on the page they  should be not a page letting them know good job. On click here to get back to creating users